data_explorer/app/config.py

```diff
@@ -1,12 +1,9 @@
 """General configuration for the app."""
 
 SESSION_STATE_VARIABLES = [
```

Comment:
We should aim to get rid of these completely. 

This is currently used to initialize all session state variables at the start of the application, to prevent errors when accessing non-initialized values. But this brings us in broken states anyway. We should make sure the flow is set up so values are only accessed after initialization, or non-initialization is handled in the accessing code.